mon-pix/app/styles/globals/_alert.scss

```
@@ -14,7 +14,8 @@
 
   &--danger {
     border: 1px solid $pix-error-70;
-    background-color: $pix-error-10;
+    background-color: $pix-error-70;
+    color: $pix-error-70;
```

Je pense que c'est plutôt
```suggestion
    background-color: $pix-error-70;
    color: $pix-error-10;
```

---

@squirrel4k je retrouve plus la doc associée

---

Pour info il existe un composant [PixBanner](https://ui.pix.fr/?path=/docs/notification-banner--error#error)

---

Yes il faudrait qu'on intègre le composant du DS ! Je crée un ticket pour ça :)

---

https://github.com/1024pix/pix/pull/4632